env/context.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package env
+
+import (
+	"net"
+	"sync"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/go-framed-msgpack-rpc"
+)
+
+// Context is an implementation for libkbfs.Context
```

Comment:
Do you mean `libkb.Context`?  If so, can you please put a `var _ libkb.Context = (*Context)(nil)` somewhere in this file?